src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Resource Attributes fixed to allow the values of key-value pairs to be of any
+  object type, matching the behavior of Activity's SetTag method.
```

Comment:
nit: move this to line 17, since this happened after all the others. Add the PR just like the others